core/amber/src/main/python/core/runnables/test_main_loop.py

```diff
@@ -678,7 +681,7 @@ def send_pause(
             payload=ControlPayloadV2(
                 return_invocation=ReturnInvocationV2(
                     original_command_id=command_sequence,
-                    control_return=ControlReturnV2(worker_state=WorkerState.PAUSED),
+                    control_return=ControlReturnV2(),
```

Comment:
how come this payload is removed from the `ControlReturn`?

Reply:
I'll fix this in the different PR